studio/data/database-triggers/database-trigger-update-transaction-mutation.ts

```diff
@@ -40,8 +40,8 @@ export async function updateDatabaseTrigger({
   updatedTrigger,
 }: DatabaseTriggerUpdateVariables) {
   const sql = getDatabaseTriggerUpdateSQL({ originalTrigger, updatedTrigger })
-  await executeSql({ projectRef, connectionString, sql })
-  return updatedTrigger
+  const { result } = await executeSql({ projectRef, connectionString, sql })
+  return result;
```

Comment:
can you revert these changes?

Reply:
"Sorry for these minor issues on my part, @MildTomato